streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -224,7 +228,7 @@ public void shouldPutFetchRangeFromCache() {
         cachingStore.put(bytesKey("a"), bytesValue("a"));
         cachingStore.put(bytesKey("b"), bytesValue("b"));
 
-        final KeyValueIterator<Windowed<Bytes>, byte[]> iterator = cachingStore.fetch(bytesKey("a"), bytesKey("b"), 10, 10);
+        final KeyValueIterator<Windowed<Bytes>, byte[]> iterator = cachingStore.fetch(bytesKey("a"), bytesKey("b"), ofEpochMilli(10), ofMillis(0));
```

Comment:
change in behavior (similar below in this test)

Reply:
Maybe "change in behavior"  is not the correct term to use here. I wanted to point out, that we need to pass in different "numbers" too preserve the behavior because the semantic of the second number changed from "point-in-time" to "duration".